teams.md/src/components/include/in-depth-guides/user-authentication/python.incl.md

```diff
@@ -85,3 +85,23 @@ async def handle_signout_message(ctx: ActivityContext[MessageActivity]):
     await ctx.sign_out()
     await ctx.send("You have been signed out!")
 ```
+
+<!-- regional-bot -->
+## Regional Configs
+You may be building a regional bot that is deployed in a specific Azure region (such as West Europe, East US, etc.) rather than global. This is important for organizations that have data residency requirements or want to reduce latency by keeping data and authentication flows within a specific area.
+
+To properly configure your regional bot with ATK, you will need to make a few updates. This example uses West Europe, but follow the equivalent for other regions.
+
+1. In `azurebot.bicep`, replace all `global` occurrences to `westeurope`
+2. In `manifest.json`, in `validDomains`, `*.botframework.com` should be replaced by `europe.token.botframework.com`
+3. In `aad.manifest.json`, replace `https://token.botframework.com/.auth/web/redirect` with `https://europe.token.botframework.com/.auth/web/redirect`
+4. You will also need to update the setup of your App instance to add in your Oauth URL.
+
+```python
+app = App(
+    default_connection_name='graph',
+    api_client_settings=ApiClientSettings(
+        oauth_url=os.environ.get("OAUTH_URL") # Value would be "https://europe.token.botframework.com"
```

Comment:
You don't need this at all right? It should be updates in the .env file or wherever they set their environment variables

Reply:
I think it's still good to showcase here so they know what the value should look like